app/assets/stylesheets/app/components/deprecated_card.scss

```diff
@@ -22,19 +22,19 @@
       opacity: 0.05;
     }
   }
+}
 
-  .deprecated-card-inner {
-    position: absolute;
-    display: block;
-    height: 100%;
-    width: 100%;
-    background-size: cover !important;
-    background-position: center !important;
-    opacity: 1;
-    @include transition-property(opacity);
-    @include transition-duration($transition-out-duration);
-    @include transition-timing-function($transition-function);
-  }
+.deprecated-card-inner {
+  position: absolute;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered background-position, background-size, display, height, opacity, position, width